### PR TITLE
Reformat blog and documentation setup

### LINKED ISSUE #1824: Reformat documentation and blog setup
### What should be done?

* [x] FerretDB logo image (on light theme) in documentation and blog should be dark blue. 
* [x] Order of the documentation should be modified to be more readable (For example- References should appear last on the sidebar menu list)
* [x] Add navbar and footer link to FerretDB website on blog and documentation
* [x] Check social card display for blogs

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #1824.

## Readiness checklist

* [ ] I added unit tests for new functionality or bug fixes.
* [ ] I added integration tests for new functionality or bug fixes.
* [ ] I added compatibility tests for new functionality or bug fixes.
* [ ] I made spot refactorings.
* [x] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by rumyantseva — APPROVED** [review]
LGTM

File: website/blog/2021-12-07-mongodb-compatibility.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 slug: mongodb-compatibility-whats-really-important
 title: "MongoDB Compatibility - What’s Really Important ?"
 author: Peter Farkas
-image: ../static/img/blog/blog-img.png
+image: /img/blog/blog-img.png
```
Comment (chilagrow) [inline website/blog/2021-12-07-mongodb-compatibility.md:5]: Just noticed that this blog is odd one out without `description`.

**Review by chilagrow — COMMENTED** [review]
Minor comment :hugs:

File: website/blog/2021-11-05-mangodb-overwhelming-enthusiasm.md [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
+description: Earlier this week, we unveiled our vision for MangoDB, the Truly Open Source MongoDB Replacement, which allows developers to use the powerful MongoDB interface while storing data in PostgreSQL.
+image: /img/blog/home-img.jpg
 tags: [FerretDB, open source database]
 ---
 
-![MangoDB: Enthusiasm for Truly Open Source MongoDB Replacement](../static/img/blog/home-img.jpg)
+Earlier this week, we unveiled our vision for MangoDB, the Truly Open Source MongoDB Replacement, which allows developers to use the powerful MongoDB interface while storing data in PostgreSQL.
 
-<!--truncate-->
+![MangoDB: Enthusiasm for Truly Open Source MongoDB Replacement](/img/blog/home-img.jpg)
```
[the remark this answers is not in the document]
Comment (Fashander) [reply website/blog/2021-11-05-mangodb-overwhelming-enthusiasm.md:12]: Although they work well as just images inside the article itself., the issue with using relative paths is how Docusaurus resolves them in metatags, and social media cards also.  For instance, here is how it currently displays the blogpost images:
Screenshot 2023-01-23 at 09 35 10

And here is how it'll look now:
Screenshot 2023-01-23 at 09 38 19

**Review by chilagrow — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
If the changes solve the problem, I'm glad to approve!

**Review by w84thesun — APPROVED** [review]

**Review by ptrfarkas — APPROVED** [review]
LGTM

File: website/blog/2021-11-05-mangodb-overwhelming-enthusiasm.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ptrfarkas) [reply website/blog/2021-11-05-mangodb-overwhelming-enthusiasm.md:12]: Priority now is to find a way for Twitter to pick up the image

**Pull request merged**